### PR TITLE
[AI Deploy - New guide] - How to deploy Whisper 

### PR DESCRIPTION
**Pull request by MathieuBsqt**
A new guide dedicated to the AI Deploy product, which explains how to deploy the Whisper model on our infrastructure. 
Whisper is a popular speech to text model, developed by OpenAI.

FYI:

🇬🇧 only the GB `guide.en-gb.md` guide has been created
🔗 `index.md` has been updated
✅ `meta.yaml` file has been created

🇫🇷 French translations:

title: AI Deploy - Tutoriel - Déployer le modèle Whisper
excerpt: Comment déployer Whisper, le modèle de reconnaissance de la parole d'OpenAI

### CODE REVIEW

File: pages/public_cloud/ai_machine_learning/deploy_tuto_17_streamlit_whisper/guide.en-gb.md [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+---
+
+> [!primary]
+>
+> AI Deploy is covered by **[OVHcloud Public Cloud Special Conditions](https://storage.gra.cloud.ovh.net/v1/AUTH_325716a587c64897acbef9a4a4726e38/contracts/d2a208c-Conditions_particulieres_OVH_Stack-WE-9.0.pdf)**.
+>
+
+## Introduction
+
+OpenAI's Whisper large-v3 is the latest iteration of the open-source Whisper Automatic Speech Recognition (ASR) model.
```
Comment (eleapttn) [inline pages/public_cloud/ai_machine_learning/deploy_tuto_17_streamlit_whisper/guide.en-gb.md]: I would have put a link on **OpenAI's Whisper large-v3** to model information

File: pages/public_cloud/ai_machine_learning/deploy_tuto_17_streamlit_whisper/guide.en-gb.md [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+
+Released on November 6, 2023, Whisper large-v3 has gained recognition in various benchmarks as the top-performing automatic speech recognition (ASR) model. This is why it represents the state of the art in the ASR field. More generally, Whisper has become a powerful tool for transcribing speech in approximately 100 languages, including non-english speech transcription and translation into English.
+
+![Overview](images/speech-to-text.png){.thumbnail}
+
+## Objective
+
+In this tutorial, we will guide you through the process of deploying the Whisper large-v3 model in a simple application on AI Deploy for production use. 
+
+Unlike a locally hosted application, using AI Deploy offers extremely fast inference since it is deployed on powerful resources (GPUs), and other features like sharing capabilities. You will also learn how to build and use a custom Docker image as part of the deployment process.
```
Comment (eleapttn) [inline pages/public_cloud/ai_machine_learning/deploy_tuto_17_streamlit_whisper/guide.en-gb.md:24]: I would have added the link of the guide about "How to create custom Docker image" on custom **custom Docker image**

File: pages/public_cloud/ai_machine_learning/deploy_tuto_17_streamlit_whisper/guide.en-gb.md [hunk 3]
```diff
@@ -0,0 +1,306 @@
...
+> The second one indicates where the model will be saved. **It must be the same place** as where you mounted the first volume. This will allow the model to be backed up in the bucket, and not in the job's ephemeral storage. Here, we use the `/workspace/whisper-model` path.
+>
+> Moreover, you may have to **change the bucket name and region** of the last `--volume` parameter, based on the name you gave to it and the region where you created it.
+
+The job will then be launched. It will take a few minutes for the two volumes to be added to the job, the environment to be installed and the Whisper model to be downloaded and synchronized with your bucket (`FINALZING` status).
+
+If you have configured your volumes correctly with the right permissions, and given the right paths to the python script, then you should see your Whisper model in your bucket. This can be checked on the Control Panel, or with the CLI with the following command, that will list all the objects of your bucket:
+
+```bash
+`ovhai bucket object list <bucket_name>@<region>`
```
Comment (eleapttn) [inline pages/public_cloud/ai_machine_learning/deploy_tuto_17_streamlit_whisper/guide.en-gb.md]:
```suggestion
`ovhai bucket object list <bucket_name>@<alias>`
```
You could change `region` by `alias` or `datastore_alias`

**Comment by MathieuBsqt** [comment]
Ready for proofreading 😁

**Comment by OVH-Team-Guides** [comment]
Thank you for your contribution @MathieuBsqt

**Pull request merged**